src/analysis/special_functions/exp_log.lean

```diff
@@ -475,6 +475,18 @@ end
 lemma log_nonpos (hx : 0 ≤ x) (h'x : x ≤ 1) : log x ≤ 0 :=
 (log_nonpos_iff' hx).2 h'x
 
+lemma log_inj_pos {x y : ℝ} (x_pos : 0 < x) (y_pos : 0 < y)
+  (h : real.log x = real.log y) : x = y :=
```

Comment:
You're inside the `real` namespace, so you don't need the `real.` here.
```suggestion
lemma log_inj_pos {x y : ℝ} (x_pos : 0 < x) (y_pos : 0 < y) (h : log x = log y) : x = y :=
```
Same below